src/main/java/com/sintern/repository/StudentRepository.java

```diff
@@ -11,4 +11,6 @@
     boolean existsByEmail(String email);
 
     boolean existsByEmailAndPassword(String email, String password);
+
+    Student findStudentById(UUID studentID);
```

Comment:
Isn't this the same thing with findById already defined by JpaRepository?

Reply:
findById returns an Optional<Student> rather than a Student, and it's more code to refactor by doing this than just make a new find method